load-testing/tests/init_load_test.go

```diff
@@ -11,12 +11,19 @@ import (
 )
 
 var (
-	flagLogLevel string
-	logger       polylog.Logger
+	flagLogLevel         string
+	flagManifestFilePath string = "../../loadtest_manifest.yaml"
+	logger               polylog.Logger
 )
 
 func init() {
 	flag.StringVar(&flagLogLevel, "log-level", "", "Specifies the log level for the runner")
+	flag.StringVar(
+		&flagManifestFilePath,
+		"manifest",
+		"../../loadtest_manifest.yaml",
```

Comment:
You assigned a default but not using it.

1. Move the default to a constant
2. Don't assign the default to the var
3. Use the constant here

Reply:
@Olshansk @red-0ne I would argue that it's be more idiomatic to construct the path with `filepath.Join`. This would also be OS-agnostic.

Reply:
I can't argue with what @bryanchriswhite is arguing for.